uint64 index
Position target_pos

# WaybillStatus
uint8 status
uint8 BILL_UNKOWN = 0
uint8 NOT_STARTED = 1
uint8 DELIVERY = 2
uint8 OVER = 3
uint8 BILL_ERROR = 5

DynamicPosition dynamic_position
